src/components/LocaleSelect.astro

```diff
@@ -0,0 +1,27 @@
+---
+import Select from 'fulldev-ui/components/Select.astro'
```

Comment:
Add a type, you can use a modified version of ComponentProps<Select>

You should omit values not needed, and add values that are unique for the LocaleSelect

Reply:
There are no unique values